package-lock.json

```diff
@@ -65,7 +65,7 @@
         "lodash": "4.17.21",
         "lottie-react-native": "6.4.1",
         "mapbox-gl": "^2.15.0",
-        "onfido-sdk-ui": "13.6.1",
+        "onfido-sdk-ui": "^14.15.0",
```

Comment:
@marcochavezf I see a conflict about the version in package.json and pack-lock.json
package.json: 14.15.0
package-lock.json: ^14.15.0

It doesn't cause any problem, but it's generally a good practice to keep the version specifications consistent between package.json and package-lock.json

Reply:
Yeah, good call. I agree with the consistency in versions